features/admin_center_dashboard.feature

```diff
@@ -123,8 +123,8 @@ Feature: Dashboard
     And I follow "Manage Users"
     Then I should see the following table rows:
       | Affiliate Manager affiliate_manager@fixtures.org            |
-      | Jane Admin admin2@fixtures.gov (Pending email verification) |
-      | John Admin admin1@fixtures.gov (Pending email verification) |
+      | Jane Admin admin2@fixtures.gov |
```

Comment:
These lines should still include the approval status, to confirm that the expected info appears on the page:
```
| Jane Admin admin2@fixtures.gov (Pending approval) |
| John Admin admin1@fixtures.gov (Pending approval) |
```

Reply:
Per our discussion, the expected status for each of these users is "Approved".

Reply:
Per our discussion this remains blank does not show (Approved) when approved.